tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# mbed-build.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2017, ARM Limited, All Rights Reserved
+#
+# Purpose
+#
+# To run test builds of the mbed OS module for all supported targets.
+
+set -eu
+
+MBED_APP='mbed-os-example-tls'
+CPUS=$(grep -c ^processor /proc/cpuinfo)
+
+check_tools()
+{
+    for TOOL in "$@"; do
+        if ! `hash "$TOOL" >/dev/null 2>&1`; then
+            echo "$TOOL not found!" >&2
+            exit 1
+        fi
+    done
+}
+
+mbed_build()
+{
+    TARGET=$1
+    TOOLCHAIN=$2
+
+    echo; echo "*** $TARGET (release $TOOLCHAIN) ***"
```

Comment:
If you need more features than what traditional echo offers, I would suggest `printf`